airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -278,14 +281,26 @@ private Job getJob(DSLContext ctx, long jobId) {
   }
 
   @Override
-  public List<Job> listJobsWithStatus(ConfigType configType, JobStatus status) throws IOException {
+  public List<Job> listJobsWithStatus(JobStatus status) throws IOException {
+    return listJobsWithStatus(Sets.newHashSet(ConfigType.values()), status);
+  }
+
+  @Override
+  public List<Job> listJobsWithStatus(Set<ConfigType> configTypes, JobStatus status) throws IOException {
+    String[] clauses = configTypes.stream().map(type -> "CAST(config_type AS VARCHAR) = ? ").toArray(String[]::new);
```

Comment:
Same here. there are example in this file of how to query for the replication jbos